server/proxy-request.js

```diff
@@ -25,10 +24,39 @@ let passThroughHeaders = function(incomingHeaders) {
 
 
 module.exports = function(url, req) {
-  // support GET or HEAD requests
-  let method = req.method === 'HEAD' ? 'HEAD' : 'GET';
+  // support GET or HEAD or POST requests
+  let methods = ['GET', 'HEAD', 'POST'];
+  let method  = null;
+  if (methods.indexOf(req.method) > -1) {
+    method = req.method;
+  } else {
+    method = 'GET';
+  }
   let externalReqHeaders = passThroughHeaders(req.headers);
 
+  if (method !== 'POST') {
+    externalReqHeaders = u.omit(externalReqHeaders, 'user-agent');
+  }
```

Comment:
> 'user-agent' header is removed at https://github.com/afeld/jsonp/blob/master/server/proxy-request.js#L16

Hmm, you're right...not sure why I did that. Maybe just to be more up-front about the fact that the request is being made from a proxy? We can remove it in this pull request or deal with it separately, but in either case, I think these three lines can be deleted.